osmaxx/api_client/conversion_api_client.py

```diff
@@ -70,6 +71,15 @@ def create_job(self, parametrization, callback_url):
         response = self.authorized_post(url='conversion_job/', json_data=json_payload)
         return response.json()
 
+    def get_result_file(self, job_id):
+        download_url = self._get_result_file_url(job_id)
+        response = self.authorized_get(download_url)
+        return ContentFile(response.content)
```

Comment:
ContentFile keeps the file in memory. They can be quite big, especially since we will be allowing bigger areas to be cut out. I'd favor a solution that doesn't write the file into memory.